Library/Koala/Koala.swift

```diff
@@ -1591,6 +1591,25 @@ public final class Koala {
     self.track(event: "Changed Live Stream Orientation", properties: props)
   }
 
+  public func trackClosedLiveStream(project: Project,
+                                    liveStream: Project.LiveStream,
+                                    startDate: Date,
+                                    endDate: Date,
+                                    refTag: RefTag) {
+    let context = stateContext(forLiveStream: liveStream)
+
+    let duration = AppEnvironment.current.calendar
+      .dateComponents([.second], from: startDate, to: endDate).second ?? 0
```

Comment:
you should be able to just subtract the dates' `timeIntervalSince1970` and not use the calendar